server/service/src/auth.rs

```diff
@@ -203,6 +204,14 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
             PermissionDSL::HasPermission(Permission::Report),
         ]),
     );
+
+    map.insert(
+        Resource::QueryLog,
+        PermissionDSL::And(vec![
+            PermissionDSL::HasStoreAccess,
```

Comment:
Why do you need store access? The logs() method doesn't filter by store_id either. @andreievg  think the store_id parameter can be removed from logs() or is there anything planned to log store id in the future? think it was not needed, right?